.github/workflows/rworkflows.yml

```diff
@@ -23,17 +27,17 @@ jobs:
       matrix:
         config:
         - os: ubuntu-latest
-          bioc: devel
+          bioc: release
```

Comment:
Remember to switch these back to devel. The Docker-based runs are currently failing due to this issue (probably and hopefully): https://github.com/markrobinsonuzh/TreeSummarizedExperiment/issues/88

As the development should be done against devel version, the testing should also be done in devel version

Reply:
That was a change i pulled from devel due to #99. I changed it back to devel as requested: windows checks stopped failing, ubuntu and mac checks should be resolved when the issue in TreeSummarizedExperiment is fixed.